scripts/update_versions.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
Same as before, probably a good time to move this to a JS file (maybe with tests).